arviz/tests/conftest.py

```diff
@@ -22,6 +22,7 @@ def save_figs(request):
         print("Directory {} created".format(fig_dir))
 
         # Clear all files from the directory
+        # Does not touch directories
```

Comment:
I'm being exceptionally nitpicky here but "touch" seems ambigious. I think `does not alter or delete directories` would be more clear.

Not required to change, just my backseat coding :)

Reply:
Good point. (It was just a quick fix to fix coveralls report). I will change it to

    Does not alter or delete directories